src/main/resources/liquibase/crop_changelog/22_1_0.xml

```diff
@@ -39,5 +40,83 @@
 			DROP COLUMN name;
 		</sql>
 	</changeSet>
+	
+	<changeSet author="rowena" id="v22.1.0-3">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+			SELECT COUNT(*)>1
+			FROM location l1
+				INNER JOIN location l2
+				WHERE l1.locid  &lt;l2.locid 
+				AND l1.lname = l2.lname AND l1.ltype = l2.ltype AND l1.cntryid = l2.cntryid 
+				AND l1.locid = l1.snl1id;
+			</sqlCheck>
+		</preConditions>
+		<comment>Update location duplicates</comment>
+      <sql>
+      UPDATE germplsm g INNER JOIN location l1 INNER JOIN location l2
+		  SET glocn = l1.locid
+		  WHERE g.glocn = l2.locid AND l1.locid &lt; l2.locid AND l1.lname = l2.lname 
+		  AND l1.ltype= l2.ltype AND l1.cntryid = l2.cntryid AND l1.locid=l1.snl1id;
+		
+		 UPDATE atributs a INNER JOIN location l1 INNER JOIN location l2
+		 SET a.alocn = l1.locid 
+		 WHERE a.alocn = l2.locid AND l1.locid &lt; l2.locid AND l1.lname = l2.lname 
+		 AND l1.ltype= l2.ltype AND l1.cntryid = l2.cntryid AND l1.locid=l1.snl1id;
+
+     UPDATE atributs_invalid_atype_bkp a INNER JOIN location l1 INNER JOIN location l2
```

Comment:
@mchabito please remove updating atributs_invalid_atype_bkp, we are adding a changeset to drop this table as it is not further needed